<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MacBookAir.fios-router.home" failures="1" tests="1" name="apiRunner.TestRunner" time="0.255" errors="0" timestamp="2024-11-22T12:13:34 EST" skipped="0">
  <testcase classname="apiRunner.TestRunner" name="runScenario" time="0.255">
    <failure message="expected [404] but found [200]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [404] but found [200]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at generics.ApiUtils.validateStatusCode(ApiUtils.java:27)
at stepDefinitions.class2Steps.admin_receives_status_for_updateClass(class2Steps.java:397)
at âœ½.Admin receives 404 "Not Found" Status for updateClass(file:///Users/mathumathibalakrishnan/eclipse/Mathumathi_2024/Team2_ChangeMakers_RestAssured/src/test/resources/features/class2.feature:177)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- apiRunner.TestRunner -->
